cli/commands/genesis/storage_test.go

```diff
@@ -37,10 +37,9 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+//nolint:paralleltest // t.TempDir() do not allow parallelism
 func TestSetDepositStorageCmd(t *testing.T) {
-	t.Parallel()
```

Comment:
Changes in the testing codebase are all related to `usetesting` replacement of `tenv`